scripts/generate_json_docs.py

```diff
@@ -290,10 +291,11 @@ def build_type(type_id, title, contents):
 
 def clean_source_path(source):
     source_path = ''
-    if 'gcloud-python' in source:
-        source_path = source.split('gcloud-python')[1]
+    for path in sys.path:
+        if path in source:
+            source_path = source.replace(path, '')
 
-    return source_path[1:]
+    return source_path
```

Comment:
So the issue comes between generating the docs locally on a system and on travis. The paths are slightly different. The goal is to trim it down to `/gcloud/datastore/client.py`.   Since the installed path varies from system to system I can't hardcode it.

I could just get the `gcloud` absolute path though and use that.
`path = os.path.abspath(gcloud.__file__)`